spec/models/like_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```

Comment:
- [ ] Kindy check on your like model, I encountered the error below when running the tests:
![image](https://user-images.githubusercontent.com/61408860/190589038-e5d73b7d-8abb-4025-96f5-7ac8adf208d8.png)